velox/docs/configs.rst

```diff
@@ -215,6 +215,12 @@ Spilling
      - boolean
      - true
      - When `spill_enabled` is true, determines whether HashAggregation operator can spill to disk under memory pressure.
+   * - partial_aggregation_spill_enabled
+     - boolean
+     - false
+     - When `spill_enabled` is true, determines whether the partial phase of HashAggregation operator can spill to disk under memory pressure.
```

Comment:
Let's explain when it could be beneficial to enable this? E.g. much fewer reducers, slow or inefficient shuffle, anything else?

Reply:
Can we just describe the result when turning the option on? For example, just note user the size of data emitted from partial aggregation would be reduced as much as possible if setting to `true`. Users will know it should be on if they rely on inefficient shuffle or have other specific reasons.